mgmt/LocalManager.cc

```diff
@@ -122,9 +122,9 @@ LocalManager::rollLogFiles()
 }
 
 void
-LocalManager::hostStatusSetDown(const char *name)
+LocalManager::hostStatusSetDown(const char *req, int len)
```

Comment:
Still len?

Reply:
Should arguments be renamed to indicate it is marshaled data?